lib/rchess.rb

```diff
@@ -113,6 +113,40 @@ def get_valid_pawn_moves(s_rank, s_file, op)
     end
   end
   # todo en passent
+
+  ##### En Passant still does not capture the piece #####
+
+  #white:
+  if s_rank == 4
+    #checking left:
+    if @board[s_rank, s_file - 1] == "♟"
```

Comment:
The logic looks good, but a few things:
* Looks like you're working off an older version of master, and have some conflicts. Pull master and resolve conflicts.
* `@board[rank, file]` should be replaced with `get_piece(rank, file)` (OO abstractions coming soon for easier testing)
* The `get_valid_pawn_moves` method is intended to be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) - try to get these changes to use less repeated code.